src/install/aim/agent-lang/python-config.mdx

```diff
@@ -0,0 +1,71 @@
+---
+componentType: default
+headingText: Configure the Python agent
+---
+
+You installed the Python agent in the last step, but you still need to make configurations so it recognizes and collects AI-specific data. For AI data, you need to enable the `ai_monitoring` and `ai_monitoring.streaming` flags. Both prompt the Python agent to expose AI data within your app data. 
+
+1. We recommend applying these configurations from `newrelic.ini` config file. If you have a more complex set up with multiple environments, you can configure with environment variables.
+
+    <CollapserGroup>
+        <Collapser
+            id="config-file"
+            title="Enable with config file"
+        >
+
+    ```yaml
+    ai_monitoring.enabled = true
+    ai_monitoring.streaming = true
+    ```   
+        </Collapser>
+        <Collapser
+            id="config-enviro"
+            title="Enable with environment variables"
+        >
+
+    ```bash
+    NEW_RELIC_AI_MONITORING_ENABLED=true
+    NEW_RELIC_AI_MONITORING_STREAMING=true
+    ```
+
+        </Collapser>
+    </CollapserGroup>
+
+3. One instance of the Python agent sends a maximum of 100,000 machine learning (ML) events per minute. Once the agent hits that maximum, it samples your ML events, giving a general representation of events across the harvest cycle. 
```

Comment:
What does the harvest cycle refer to here? I have an idea, but want to hear it in your words @RyanKadri